### PR TITLE
Add Shareable URLs docs for Kedro-Viz

### PR DESCRIPTION
**Pull request by tynandebold**
## Description

Resolves https://github.com/kedro-org/kedro-viz/issues/1530

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [x] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

**Comment by stichbury** [comment]
I made a commit to the branch to resolve the Vale linter comments and make a few changes. I changed the page name/title to reflect more about the sharing aspect, see what you think and revert to your choice if you think it's better.

**Comment by tynandebold** [comment]
> I made a commit to the branch to resolve the Vale linter comments and make a few changes. I changed the page name/title to reflect more about the sharing aspect, see what you think and revert to your choice if you think it's better.

Thank you! I like yours more :)

**Review by stichbury — APPROVED** [review]
I am really happy with these, thanks @tynandebold !

File: docs/source/visualisation/share_kedro_viz.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Share a project's Kedro-Viz
+
+```{note}
+Kedro-Viz sharing was introduced in version 7.0.0.
+```
+
+This page describes how to deploy the Kedro-Viz for your project so you can share it with others. It uses the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) as an example.
```
Comment (stichbury) [inline docs/source/visualisation/share_kedro_viz.md:7]:
```suggestion
This page describes how to host the Kedro-Viz for your project so you can share it with others. It uses the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) as an example.
```

Comment (stichbury) [reply docs/source/visualisation/share_kedro_viz.md:7]: Thanks @rashidakanchwala this is definitely a better term/description here.

File: docs/source/visualisation/share_kedro_viz.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Once that's completed, you'll need to set your AWS credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AWS_ACCESS_KEY_ID="your_access_key_id"
+export AWS_SECRET_ACCESS_KEY="your_secret_access_key"
+```
+
+For more information, see the official AWS documentation about [how to work with credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
+
+## Deploy the project
```
Comment (stichbury) [inline docs/source/visualisation/share_kedro_viz.md:51]:
```suggestion
## Host the project
```

File: docs/source/visualisation/share_kedro_viz.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+You're now ready to deploy your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
```
Comment (stichbury) [inline docs/source/visualisation/share_kedro_viz.md:53]:
```suggestion
You're now ready to host your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
```

File: docs/source/visualisation/share_kedro_viz.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+```bash
+kedro viz
+```
+
+Click the **Deploy and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
+
+Once those two details are complete, click **Deploy**. A hosted, shareable URL will be returned to you after the deploy completes.
```
Comment (stichbury) [inline docs/source/visualisation/share_kedro_viz.md:61]: Ah, only problem with "host" is that the UI has "deploy" in it.

**Comment by tynandebold** [comment]
@stichbury I've made changes to make things consistent with what we were all discussing earlier, removing `deploy` in favor of `publish`. 

Have also changed the page title from `Share a project’s Kedro-Viz` to `Publish and share Kedro-Viz`.

Will merge tomorrow if you're ok with it!

**Comment by stichbury** [comment]
I put my comment in the issue rather than here. https://github.com/kedro-org/kedro-viz/issues/1530#issuecomment-1738749496

**Review by noklam — APPROVED** [review]

**Pull request merged**